test/specs/real-world/fetch-api-list.js

```diff
@@ -1,65 +0,0 @@
-/* eslint-disable no-param-reassign */
```

Comment:
Removed this in favor of `bin/refresh-real-world-apis` and running the tests off a pre-cached dataset. It was getting a bit annoying pushing up changes to CI and having them fail because specs got bumped on apis.guru and I had to spend more time re-running the real-world test suite to exclude known errors.